### ISSUE TITLE
Link to a mercurial clone of the project

### ISSUE THREAD

**Issue by gjnoonan** [ananthakumaran/monky#56]
I understand the reasoning behind this plugin being hosted in git and on github.
However I think that since it is indeed a plugin for interacting with mercurial
then we should at least have a version in a mercurial repo.

I have a copy (via hg-git) up on bitbucket, is there an "official" one?

**Comment by ananthakumaran** [comment]
No, there is no official hg mirror anywhere. I know it's ironic and there were heated discussions about it in emacs irc channel. As I have stated in other places, I prefer git over hg (not because one is superior over other but because of the universality of git). This plugin was written(I had to work on a hg repo) because I found the hg plugins inadequate after having used magit for sometime.

I am not against hosting hg mirror if somebody is willing to set it up and maintain it. Though I don't see any big advantage with the mirror as the emacs users are mostly going to install it via melpa and contributors who prefer hg can use hg-git? I have never used it myself.
